src/app/test_executive/dune

```diff
@@ -1,66 +1,66 @@
 (executable
  (name test_executive)
  (libraries
-  ;; opam libraries
-  async_kernel
-  async
-  core
-  uri
-  yojson
-  core_kernel
-  cmdliner
-  base.base_internalhash_types
-  base.caml
-  async_unix
-  unsigned_extended
-  stdio
-  sexplib0
-  ;; local libraries
-  mina_wire_types
-  with_hash
-  data_hash_lib
-  kimchi_backend
-  kimchi_backend.pasta
-  kimchi_backend.pasta.basic
-  pickles
-  pickles_types
-  random_oracle_input
-  genesis_constants
-  bash_colors
-  integration_test_lib
-  signature_lib
-  mina_signature_kind
-  mina_base
-  mina_stdlib
-  mina_transaction
-  file_system
-  currency
-  mina_runtime_config
-  secrets
-  integration_test_cloud_engine
-  integration_test_local_engine
-  mina_generators
-  logger
-  random_oracle
-  mina_numbers
-  transaction_snark
-  snark_params
-  pickles.backend
-  pipe_lib
-  mina_base.import
-  key_gen
-  integers
-  user_command_input
-  participating_state
-  graph_algorithms
-  visualization
-  sgn
-  zkapp_command_builder
-  network_pool
-  zkapps_examples
-  cache_dir
-  snarky.backendless)
- (instrumentation
-  (backend bisect_ppx))
- (preprocess
-  (pps ppx_mina ppx_jane ppx_deriving_yojson ppx_mina ppx_version)))
+   ;; opam libraries
```

Comment:
The original PR only adds `protocol_version` to this list. It would be better to do the same here, in the new format.